### PR TITLE
Icons updated in 2019 statistics section

### LINKED ISSUE #74: Stats Icons in register page are less semantic
**Describe the problem**
Icons present in the register page of the website are very less semantic when it comes to the text written below them.

**Proposed fix/solution**
Those icons need to be changed in accordance to the entity mentioned there.

**Screenshots/Visuals**
icons

### PR DESCRIPTION
**Pull request by yash0501**
Icons updated in 2019 statistics section ( Solved issue #74 )

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[1227/155125.116:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (s-ayush2903) [inline debug.log]: We don't need to commit this file

**Review by s-ayush2903 — CHANGES_REQUESTED** [review]
Change the projects icon to the repo icon that github uses

**Comment by s-ayush2903** [comment]
Now it looks a weird, it has an odd size which makes it look different from the other icons, can you fix it. I'm talking about the length of the projects cards that is lesser than that of others. Have a look. You may check by the deployment made by vercel to see it
|Uneven card Size|
|--|
|weird|

**Review by s-ayush2903 — APPROVED** [review]
Good Work and thanks for making the changes quickly :+1:

**Comment by yash0501** [comment]
Thank you for making me aware of the bugs in PR.

**Comment by s-ayush2903** [comment]
@yash0501 can you please update the icon for PRs, it would be better if we show the icon that github uses for showing the _merged_ PRs, means the one which you can see after your PR has been merged up here. I don't think you would need five points for it

**Pull request merged**